From 481a7da59f20b9c1322a5d26a9bce3e94f04443d Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Fri, 17 Jul 2026 11:09:25 +0200
Subject: [PATCH] doc: nft.8: Fix markup in ct expectation synopsis

JIRA: https://issues.redhat.com/browse/RHEL-190549
Upstream Status: nftables commit ef659df2a45c38ad18f703febeae8c4febf9dd04

commit ef659df2a45c38ad18f703febeae8c4febf9dd04
Author: Phil Sutter <phil@nwl.cc>
Date:   Wed Apr 24 23:46:11 2024 +0200

    doc: nft.8: Fix markup in ct expectation synopsis

    Just a missing asterisk somewhere.

    Fixes: 1dd08fcfa07a4 ("src: add ct expectations support")
    Signed-off-by: Phil Sutter <phil@nwl.cc>

Signed-off-by: Phil Sutter <psutter@redhat.com>
---
 doc/stateful-objects.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/stateful-objects.txt b/doc/stateful-objects.txt
index 00d3c5f..5824d53 100644
--- a/doc/stateful-objects.txt
+++ b/doc/stateful-objects.txt
@@ -119,7 +119,7 @@ sport=41360 dport=22
 CT EXPECTATION
 ~~~~~~~~~~~~~~
 [verse]
-*add* *ct expectation*  ['family'] 'table' 'name' *{ protocol* 'protocol' *; dport* 'dport' *; timeout* 'timeout' *; size* 'size' *; [*l3proto* 'family' *;*] *}*
+*add* *ct expectation*  ['family'] 'table' 'name' *{ protocol* 'protocol' *; dport* 'dport' *; timeout* 'timeout' *; size* 'size' *;* [*l3proto* 'family' *;*] *}*
 *delete* *ct expectation*  ['family'] 'table' 'name'
 *list* *ct expectations*
 
